### ISSUE TITLE
Integration Test for quickstep_cli_shell

### ISSUE THREAD

**Issue by zuyu** [pivotalsoftware/quickstep#73]
We should have the integration test for `quickstep_cli_shell` in the CI test pipeline.

**Comment by pateljm** [comment]
@zuyu: Can you expand on this? Do you mean an end-to-end test? I'm working on one.

**Comment by zuyu** [comment]
@pateljm I mean, in the `Travis` CI we run `quickstep_cli_shell` with some queries, and then compare the query results with the expected ones.

**Comment by pateljm** [comment]
Good point. I am working on creating a few queries to do just that. Haven't finished it, but will try and do that soon.
